pom.xml

```diff
@@ -22,7 +22,7 @@
 
     <groupId>com.ubirch.util</groupId>
     <artifactId>ubirch-rest-akka-http-utils_${scala.major.version}</artifactId>
-    <version>0.4.2-SNAPSHOT</version>
+    <version>0.4.3-SNAPSHOT</version>
```

Comment:
I feel this is a major upgrade. maybe 1.0.0-SNAPSHOT is better?

Reply:
But wouldn't you say, that due to the postfix being used _${scala.compat.version} it is actually the same functionality version, only scala changed? No breaking changes etc. That was my thinking. But if it is best practice to make major change out of this, I could still do it. Haven't deployed the different upgrades yet..

Reply:
You're right. let's keep it 😄